### DISCUSSION TITLE
Stopped working after a while

### DISCUSSION THREAD

**Discussion by a020088** [Zeppelin500/MBusino#12] [Q&A] [answered]
I have a couple of issues but there is one which is really disappointing. 
After Start-Up everything works fine but after a while ( hours of opereation) it stopped working. It won’t acquire any data anymore. It does not crash completely because the web front end is still accessible. I have to repower and then it starts working again. Any ideas?
My second question is: how to update OTAR since you mentioned over the air update is working.

**Comment by Zeppelin500** [comment]
The second question is easy, just chose at the Arduino IDE 2 the IP in ports.

No, no idea. Did you receive any MQTT message at this time? Nothing or just no M-Bus? Or did you receive periodically the MQTT start message?
What kind of hardware do you use? What kind of slave do you use?
Wi-Fi issues?

**Comment by Zeppelin500** [comment, marked as answer]
Use the new Code of MBusinoAP, it should work better with bad connections.
